mm2src/mm2_main/tests/docker_tests/tendermint_tests.rs

```diff
@@ -1,65 +1,59 @@
 use common::{block_on, log};
 use mm2_number::BigDecimal;
+use mm2_rpc::data::legacy::OrderbookResponse;
 use mm2_test_helpers::for_tests::{atom_testnet_conf, disable_coin, disable_coin_err, enable_tendermint,
                                   enable_tendermint_token, enable_tendermint_without_balance,
-                                  get_tendermint_my_tx_history, ibc_withdraw, iris_nimda_testnet_conf,
-                                  iris_testnet_conf, my_balance, send_raw_transaction, withdraw_v1, MarketMakerIt,
-                                  Mm2TestConf};
-use mm2_test_helpers::structs::{Bip44Chain, HDAccountAddressId, RpcV2Response, TendermintActivationResult,
-                                TransactionDetails};
+                                  get_tendermint_my_tx_history, ibc_withdraw, iris_ibc_nucleus_testnet_conf,
+                                  my_balance, nucleus_testnet_conf, orderbook, orderbook_v2, send_raw_transaction,
+                                  set_price, withdraw_v1, MarketMakerIt, Mm2TestConf};
+use mm2_test_helpers::structs::{Bip44Chain, HDAccountAddressId, OrderbookAddress, OrderbookV2Response, RpcV2Response,
+                                TendermintActivationResult, TransactionDetails};
 use serde_json::json;
+use std::collections::HashSet;
+use std::iter::FromIterator;
 
-const ATOM_TEST_BALANCE_SEED: &str = "atom test seed";
-const ATOM_TEST_WITHDRAW_SEED: &str = "atom test withdraw seed";
-const ATOM_TICKER: &str = "ATOM";
-const ATOM_TENDERMINT_RPC_URLS: &[&str] = &["https://rpc.sentry-02.theta-testnet.polypore.xyz"];
+const TENDERMINT_TEST_SEED: &str = "tendermint test seed";
+const TENDERMINT_CONSTANT_BALANCE_SEED: &str = "tendermint constant balance seed";
 
-const IRIS_TEST_SEED: &str = "iris test seed";
-const IRIS_TESTNET_RPC_URLS: &[&str] = &["http://34.80.202.172:26657"];
-
-const NUCLEUS_TESTNET_RPC_URLS: &[&str] = &["http://5.161.55.53:26657"];
+const ATOM_TENDERMINT_RPC_URLS: &[&str] = &["http://0.0.0.0:26658"];
+const NUCLEUS_TESTNET_RPC_URLS: &[&str] = &["http://0.0.0.0:26657"];
```

Comment:
Is 0.0.0.0 used as the loopback 127.0.0.1 address?
I guess this is probably non-standard.

Reply:
127.0.0.1 won't work on certain scenarios (e.g., when running from docker container https://serverfault.com/questions/1084915/still-confused-why-docker-works-when-you-make-a-process-listen-to-0-0-0-0-but-no).

Reply:
Switched to `localhost`